tests/core/gui/test_create_organization.py

```diff
@@ -391,6 +391,20 @@ def _user_widget_ready():
 
     await aqtbot.wait_until(_user_widget_ready)
 
+    def _custom_bckend_addr_widget_is_valid():
```

Comment:
This test does absolutely nothing. The dialog is started using a bootstrap address, which means `line_edit_backend_addr` will be read only, therefore `key_clicks` will not be able to change it.

This test should be put instead in `test_create_organization_custom_backend()` (L503), but it is currently marked as `skip`.  It would probably be a good thing to remove the skip and actually have a test using a custom backend addr.